terraform/helm/genesis/templates/genesis.yaml

```diff
@@ -84,7 +85,7 @@ spec:
       restartPolicy: Never
       containers:
       - name: genesis
-        image: {{ .Values.genesis.image.repo }}:{{ .Values.genesis.image.tag | default .Values.imageTag }}
```

Comment:
This would re-run genesis without restarting the validators and fullnodes to pick up the new genesis secrets actually, so you'd still be running the old network.